### DISCUSSION TITLE
Custom checkboxGroupInput

### DISCUSSION THREAD

**Discussion by yaurita** [teamalgoritma/community#403] [Q&A] [answered]
Siang. Mohon arahannya bagaimana cara untuk custom urutan isian pada checkboxGroupInput? makasi

**Comment by jafarhf1** [comment, marked as answer]
Dear @yaurita ,
Terkait melakukan custom urutan pada `checkboxGroupInput` bisa dilakukan pada parameter `choices` lalu masukkan urutan yang diinginkan seperti misalnya `choices = c("Low","Medium","High")`.

Demikian yang bisa kami sampaikan, semoga membantu.
